lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/mv/BridgeInterface.java

```diff
@@ -19,11 +19,8 @@
  */
 package eu.learnpad.mv;
 
-import javax.ws.rs.Path;
+import eu.learnpad.mv.rest.Verification;
 
-import eu.learnpad.mv.rest.DeadlockVerification;
-
-@Path("/learnpad/mv/bridge")

```

Comment:
Don't know why you removed the `Path` from here.  It's the common interface to all REST services for the Bridge.